### DISCUSSION TITLE
How do we skip packages that have circular dependencies through peerDependencies ?

### DISCUSSION THREAD

**Discussion by sharvilak11** [vercel/turborepo#287] [Help] [answered]
In our design system, we have certain components shipped as independent packages in a monorepo, but a few packages are webpack or tooling based utilities too which don't have any `build` or `dist` or any other folder. They are exposed by only `index.js` in its root. In their package.json we have peerDependencies as the main package which consists all packages and that creates circular dependencies.

How to tackle such issue ?

**Comment by jaredpalmer** [comment, marked as answer]
We could add a flag to ignore local peer dependency declarations. Open to alternative solutions

**Reply by sharvilak11** [reply]
Yes, that would be great!

**Comment by vinibgoulart** [comment]
have we made any progress?

**Comment by anthonyshew** [comment]
Going to close this since it has not seen much activity after almost two years. 😄 If you end up here in there future, please open an issue so we can track interest and reference this discussion, #287.
